### PR TITLE
Added GSoC updates

### PR DESCRIPTION
**Pull request by araza6**
Blog links for the first two weeks of our progress updated on the news page

### CODE REVIEW

File: news.html [hunk 1]
```diff
@@ -11,6 +11,20 @@ <h1 style="margin-bottom:20px">QuTiP News</h1>
...
+<div class="row" style='margin-bottom: 15px'>
+    <div class="col-md-8">
+        <p style='margin-bottom: -10px'>June 22, 2020</p>
+        <h4>Google Summer of Code Project Updates
+</h4>
+    </div>
+<div class="col-md-4" style='margin-top: 25px'>
+        <a href="https://www.binhbar.com/posts/2020/06/design-draft-data-structure-specifications/"><p>Jake's Blog Post</p></a>
+        <a href="https://sarsid.wordpress.com/2020/06/14/an-introduction-to-quantum-measurements-in-qutip/"><p>Sidhant's BlogPost</p></a>
+        <a href="https://araza6.github.io/posts/autodiff/autodiff/"><p>Asad's Blog Post</p></a>
```
Comment (nathanshammah) [inline news.html:23]:
```suggestion
        <a href="https://www.binhbar.com/tags/gsoc/"><p>Jake's Blog posts</p></a>
        <a href="https://sarsid.wordpress.com/tag/gsoc2020/"><p>Sidhant's Blog posts</p></a>
        <a href="https://araza6.github.io/posts/"><p>Asad's Blog posts</p></a>
```

File: news.html [hunk 2]
```diff
@@ -11,6 +11,20 @@ <h1 style="margin-bottom:20px">QuTiP News</h1>
 </div>
 <br>
 
+<div class="row" style='margin-bottom: 15px'>
+    <div class="col-md-8">
+        <p style='margin-bottom: -10px'>June 22, 2020</p>
+        <h4>Google Summer of Code Project Updates
```
Comment (nathanshammah) [inline news.html:17]:
```suggestion
        <h4>2020 Google Summer of Code Project Updates
```

**Comment by jakelishman** [comment]
@nathanshammah: Thanks!  I've actually got a handful of blog posts up, just numFOCUS only wanted one link.  We can either link my tag https://www.binhbar.com/tags/gsoc, or I currently have 6 posts (reverse chronological):
1. https://www.binhbar.com/posts/2020/06/design-draft-data-structure-specifications/
2. https://www.binhbar.com/posts/2020/06/design-draft-data-layer-separation/
3. https://www.binhbar.com/posts/2020/06/compiling-openmp-libraries-on-macos/
4. https://www.binhbar.com/posts/2020/06/getting-the-cython-debugger-running-on-macos/
5. https://www.binhbar.com/posts/2020/05/understanding-old-git-branches/
6. https://www.binhbar.com/posts/2020/05/welcome-to-google-summer-of-code-2020/

**Comment by nathanshammah** [comment]
@jakelishman thanks, yes I saw them, nice stuff! I suggested in the PR review to use the gsoc tag as that will capture also the future ones. Maybe adding a "qutip" label could be helpful too.

**Comment by jakelishman** [comment]
There's a QuTiP tag on my blog too - https://www.binhbar.com/tags/qutip.  I use that for stuff that's 100% related to QuTiP, which isn't all my GSoC posts, since some of those are more about getting general tooling functioning.

**Pull request merged**